quality-assurance/performance-test-checklist.md

```diff
@@ -0,0 +1,80 @@
+# Performance testing checklist
+
+## Purpose of this document
+This document lists activities that you should consider when planning performance testing
+
+## Activity list
+
+### Establish business requirements
+- What are the requirements for resilience under load (e.g. availability, responsiveness)
+- What will be typical and peak usage of the system
+- What are the requirements for scaling
+- What are typical patterns of usage (e.g. peaks, types of request)
+
+### Agree types of performance testing to carry out, and data to gather
+e.g.
+- Load testing
+- Scalability testing
+- Soak testing
+- Stress Testing
+etc.
+
+For each type of testing, work out what data must be collected to support quality judgements.
```

Comment:
not sure what you mean by "what data"

Reply:
For each type of testing, work out which data must be collected to support quality judgements.  For example, for load testing you might collect data about response times and failure rates and also data regarding SUT health such as CPU and memory usage.  For soak testing, you will additionally want to track SUT health data in more detail, looking for upward trends in things such as memory usage, database connections, and threads.